tests/e2e/lib/backup.go

```diff
@@ -145,3 +145,50 @@ func BackupErrorLogs(c *kubernetes.Clientset, ocClient client.Client, namespace
 	bl := BackupLogs(c, ocClient, namespace, name)
 	return errorLogsExcludingIgnored(bl)
 }
+
+func GetBackupRepositoryList(c client.Client, namespace string) (*velero.BackupRepositoryList, error) {
+	backupRepositoryList := &velero.BackupRepositoryList{
+		Items: []velero.BackupRepository{},
+	}
+	err := c.List(context.Background(), backupRepositoryList, client.InNamespace(namespace))
+	// if there is an error, just print, don't return the error
+	if err != nil {
+		log.Printf("error getting BackupRepository list: %v", err)
+	}
+	return backupRepositoryList, nil
+}
+
+func DeleteBackupRepository(c client.Client, namespace string, name string) error {
+	backupRepository := &velero.BackupRepository{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      name,
+		},
+	}
+	err := c.Delete(context.Background(), backupRepository)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// DeleteBackupRepositoryByRegex deletes a BackupRepository instance that matches the given regex pattern.
+func DeleteBackupRepositoryByRegex(c client.Client, namespace string) error {
```

Comment:
bad name, it should be something like `DeleteBackupRepositories` or `DeleteBackupRepositoriesInNamespace`

Reply:
agree, updating.

Reply:
yup, fixed